src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Rectangle extends FigureAbstract implements Figure {
+    private double width;
+    private double heigth;
+    public Rectangle(String color, double width, double heigth) {
+        super(color);
+        this.width = width;
+        this.heigth = heigth;
```

Comment:
There is a typo in the variable name `heigth`. It should be `height` to correctly represent the dimension of the rectangle.